docker-compose.yml

```diff
@@ -1,17 +1,21 @@
 version: "3.2"
 
 services:
-  dev:
+  node-subtensor:
     container_name: node-subtensor
-    image: paritytech/ci-linux:production
-    working_dir: /var/www/node-subtensor
+    image: opentensor/subtensor:latest
+    cpu_count: 2
+    mem_limit: 4000000000
+    memswap_limit: 8000000000
     ports:
       - "9944:9944"
+      - "30333:30333"
+      - "9933:9933"
+    expose:
+      - "9944"
+      - "30333"
+      - "9933"
     environment:
       - CARGO_HOME=/var/www/node-subtensor/.cargo
-    volumes:
-      - .:/var/www/node-subtensor
-      - type: bind
-        source: ./.local
-        target: /root/.local
-    command: bash -c "cargo build --release && ./target/release/node-subtensor --dev --ws-external"
+    command: bash -c "./node-subtensor --base-path /tmp/blockchain --chain /raw_spec.json --rpc-external --ws-external --rpc-cors all --no-mdns --ws-max-connections 10000 --in-peers 500 --out-peers 500 --bootnodes /ip4/13.58.175.193/tcp/30333/p2p/12D3KooWEXY393DjTvBip3DnZeJkSAeZrg6edxqu45ywHzzQS2BY"
```

Comment:
There is a way to write the command in multiple lines for a docker compose file. I was looking for it, but I don't find it. 
I think will be nice to have it the command in multiple lines, it can make it easier to read

Reply:
Yep it is possible to abuse the `entrypoint` key, though may still not be as pretty as either of us would hope, eg

```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 060457e..6f3fcc9 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -17,5 +17,25 @@ services:
       - "9933"
     environment:
       - CARGO_HOME=/var/www/node-subtensor/.cargo
-    command: bash -c "./node-subtensor --base-path /tmp/blockchain --chain /raw_spec.json --rpc-external --ws-external --rpc-cors all --no-mdns --ws-max-connections 10000 --in-peers 500 --out-peers 500 --bootnode
s /ip4/13.58.175.193/tcp/30333/p2p/12D3KooWEXY393DjTvBip3DnZeJkSAeZrg6edxqu45ywHzzQS2BY"
+    entrypoint:
+      - bash
+      - -c
+      - ./node-subtensor
+      - --base-path
+      - /tmp/blockchain
+      - --chain
+      - /raw_spec.json
+      - --rpc-external
+      - --ws-external
+      - --rpc-cors
+      - all
+      - --no-mdns
+      - --ws-max-connections
+      - 10000
+      - --in-peers
+      - 500
+      - --out-peers
+      - 500
+      - --bootnodes
+      - /ip4/13.58.175.193/tcp/30333/p2p/12D3KooWEXY393DjTvBip3DnZeJkSAeZrg6edxqu45ywHzzQS2BY
```

... each argument/parameter would end up on separate lines, due to `IFS` defaults, but if `=` is a valid option/value separator for Rust's argument parsing then it may be possible to pare-up things

Reply:
Thanks that's a good point! Making a subsequent PR for this